packages/material-ui-private-theming/src/index.js

```diff
@@ -0,0 +1,8 @@
+export { default as getThemeProps } from './getThemeProps';
```

Comment:
For the content of the package, did you consider keeping to its bare minimum, the React context singleton, and then, duplicate all the other modules? For instance, this one, it looks like we will want to duplicate it between the core and the legacy styled package.

Reply:
That makes sense, the only thing really required to be singleton is the React context. Let me update 👍

Reply:
Done, I've duplicated the utilities, the only things left in the new package, are the `ThemeProvider`, `useTheme` hook and the interface for the `DefaultTheme`.